GetF!t/SupportingFiles/ImageProvider.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+struct ImageProvider {
+    static let exercise: String = "exercise"
+    static let head: String = "head"
+    static let arm: String = "arm"
+    static let hands: String = "hands"
+    static let stepUp: String = "step-up"
+    static let sundawn: String = "sundawn"
+    static let battery100: String = "battery.100"
+    static let battery50: String = "battery.50"
+    static let battery0: String = "battery.0"
+    static let waveform: String = "waveform.path.ecg"
+    static let person3Fill: String = "person.3.fill"
+    static let handRaised: String = "hand.raised.fill"
+    static let personWave2Fill: String = "person.wave.2.fill"
+    static let squareGrid2x2Fill: String = "square.grid.2x2.fill"
+    static let chartBarFill: String = "chart.bar.fill"
+    static let xmark: String = "xmark"
+    static let boltFill: String = "bolt.fill"
+    static let arrowUturnUp: String = "arrow.uturn.up"
+    static let hareFill: String = "hare.fill"
+    static let sunMaxFill: String = "sun.max.fill"
+
+}
```

Comment:
toto nechám na teba či to necháš takto alebo to prerobíš podľa toho môjho štýlu 🙂  ak to chceš nechať takto tak aspoň odstráň prázdny riadok pred ukončovacou zátvorkou structu a taktiež datový typ všetkých konštánt (pretože kompilátor ho view infernúť).